src/components/container/Container.module.css

```diff
@@ -1,5 +1,5 @@
 .container {
   width: 100%;
   margin-inline: auto;
-  max-width: var(--nlportal-max-page-width);
+  max-width: var(--condution-max-page-width);
```

Comment:
There's a typo here.

Also: maybe better to create an actual design token for this component instead of a generic max-page one.

E.g.: `--conduction-container-width`.